src/test/java/core/basesyntax/service/impl/FileWriteImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWrite;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import org.junit.jupiter.api.AfterEach;
```

Comment:
Consider using more descriptive names for the test class and test methods. The current names do not clearly convey the purpose of the tests.